project/DistributionPackage.scala

```diff
@@ -271,6 +271,19 @@ object DistributionPackage {
     log.info(s"Executing $enso ${args.mkString(" ")}")
     val pb  = new java.lang.ProcessBuilder()
     val all = new java.util.ArrayList[String]()
+    val disablePrivateCheck = {
+      val findRun = args.indexOf("--run")
+      if (findRun >= 0 && findRun + 1 < args.size) {
+        val whatToRun = args(findRun + 1)
+        if (whatToRun.startsWith("test/") && whatToRun.endsWith("_Tests")) {
+          whatToRun.contains("_Internal_")
+        } else {
+          false
+        }
```

Comment:
This would be so much better to read with a simple regex.
Or
```suggestion
        whatToRun.startsWith("test/") && whatToRun.endsWith("_Tests") && whatToRun.contains("_Internal_")
```

Reply:
The first line says:
- apply this only when running _standard tests_

The second line:
- only those _named internal_ are eligible

I am not huge friend of long one liners.